### ISSUE TITLE
Incorrect item Cycle: day #

### ISSUE THREAD

**Issue by cbyekt** [jeanropke/RDR2CollectorsMap#551]
#### Found a new location? [Use this form here](https://github.com/jeanropke/RDR2CollectorsMap/issues/182)
* What day cycle it is?
* Coordinates or a clear image of the location
* Include the item name

Coordinates can be found by setting the `Show coordinates on click` to `Yes`
the last 2 days none of the items have appeared where the map shows them to be

**Comment by cbyekt** [comment]
Let me rephrase that. the coins arent showing up

**Comment by cbyekt** [comment]
rings are also not showing up

**Comment by ilinyzili** [comment]
lat: -75.78054466182577
lng: 87.95137871210159 
.
its a little east of Four of Swords - Cycle 2 
Metal Hairpin Family Heirloom - Metal Detector

**Comment by cbyekt** [comment]
Coins still arent showing.

**Comment by cbyekt** [comment]
just found 9 of swords in fort brennard in the northwest tower

**Comment by cbyekt** [comment]
just found 9 of cups at the van horn fast travel on west siDE OF RAILROAD

**Comment by cbyekt** [comment]
building on bench

**Comment by cbyekt** [comment]
coins are working today. thank you
